src/app/hero/detail/detail-container/detail-container.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+import {map} from "rxjs/operators";
+import {Hero} from "../../../model/hero";
+import {HeroDataService} from "../../hero-data.service";
+import {ActivatedRoute} from "@angular/router";
+
+@Component({
+  selector: 'mu-detail-container',
+  templateUrl: './detail-container.component.html',
+  styleUrls: ['./detail-container.component.scss']
+})
+export class DetailContainerComponent implements OnInit {
+
+  constructor(
+    private heroDataService: HeroDataService,
+    private activatedRoute: ActivatedRoute,
+  ) { }
+
+  currentHeroData!: Hero;
```

Comment:
why exclamatory?

Reply:
I  am getting compile error if i remove that